lib/modules/home/components/top_bar.dart

```diff
@@ -0,0 +1,81 @@
+part of '../home_module.dart';
+
+class TopBar extends StatelessWidget {
+  const TopBar({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    final state = context.findAncestorStateOfType<_HomeViewImplState>()!;
+
+    return SafeArea(
+      child: Container(
+          margin: const EdgeInsets.fromLTRB(20, 0, 20, 0),
+          child: Row(
+            mainAxisAlignment: MainAxisAlignment.spaceBetween,
+            children: [
+              Column(
+                mainAxisSize: MainAxisSize.min,
+                crossAxisAlignment: CrossAxisAlignment.start,
+                children: [
+                  Bone(
+                    height: 18,
+                    width: 117,
+                    child: StreamsSelector0<DateTime>.value(
+                      stream: state._dateTime,
+                      builder: (_, dateTime, __) => Text(
+                        DateFormat("EEEE, MMM d")
+                            .format(dateTime)
+                            .toUpperCase(),
```

Comment:
since you already had a state `_dateTime`, perhaps we should expose something as `formattedDateTime` which hold this date format in your HomeViewImplState instead of keeping this logic here? 🤔

Reply:
Good point! Let's me update it 👍